### ISSUE TITLE
Plugin Manager: Help button not working on gtk3

### ISSUE THREAD

**Issue by LarsGit223** [geany/geany#2151]
I just noticed that the Help button in the Plugin Manager is not working on gtk3. That means, if I click on it nothing happens. Re-compiling Geany and Geany-Plugins for gtk2 made it working again.

The other buttons (Preferences, Keybindings, Close) work without problems.

My environment: Ubuntu 18.04.2, Geany 1.36 (current master), GTK 3.22.30, GLib 2.56.4.

**Comment by b4n** [comment]
Works for me, Debian 9.9, Geany master, GTK 3.22.11, GLib 2.50.3.

**Comment by elextr** [comment]
WFM Mint 19 AKA Ubuntu 18.04, Geany 1.36 (git >= 4bceddb4), GTK 3.22.30, GLib 2.56.4

Did you realise help is only available when the plugin is selected?

**Comment by kugel-** [comment]
>Did you realise help is only available when the plugin is selected?

Also only if the selected plugin implements it.

**Comment by elextr** [comment]
> Also only if the selected plugin implements it.

Yeah that too :)

**Comment by LarsGit223** [comment]
> Did you realise help is only available when the plugin is selected?

Yes, and in that case the button is in-active.

> Also only if the selected plugin implements it.

Yes, I know that not all plugins have implemented the help feature. I tried more than one and also re-tried on one which worked on gtk2.

But here is an update:
I just found out that it only happens if I run geany as root. If I work with a normal user login under gtk3 then it works like on gtk2. So I assume this can be closed as it might be caused by some file on my machine having the wrong permission?
